styleguide/derek/view-templates/search/search.ejs

```diff
@@ -0,0 +1,54 @@
+<!--Header Medium-->
+<div class="imacBanner">
+  <div class="imacBanner-container">
+    <div class="imacBanner-textRow">
+      <div class="imacBanner-textContainer imacBanner-headlineMedium">
+        <h2 class="imacBanner-headline">Search Results</h2>
+        <span class="imacBanner-line left-top"></span>
+        <span class="imacBanner-line left-bottom"></span>
+        <span class="imacBanner-line right-bottom"></span>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="container half-padding">
+  <!-- Lead pattern -->
+  <div class="lead-wrapper">
+    <div class="lead-headerWrapper text-left">
+      <h1 class="lead-header">Search Results for: Cloud</h1>
+    </div>
+    <div class="lead-bodyWrapper text-left">
+      <p>About 6,030 results (0.31 seconds)</p>
+    </div>
+  </div>
+
+  <%- partial('_search-result') %>
+  <%- partial('_search-result') %>
+  <%- partial('_search-result') %>
+  <%- partial('_search-result') %>
+  <%- partial('_search-result') %>
+  <%- partial('_search-result') %>
+  <%- partial('_search-result') %>
+  <%- partial('_search-result') %>
+  <%- partial('_search-result') %>
+  <%- partial('_search-result') %>
+
+</div>
+
+<div class="container half-padding">
+  <div class="search-pagerWrapper col-xs-12 col-sm-8 center-block">
+    <ul class="search search-pager">
+      <li class="search-pagerItem"><a href="javascript:void(0);">1</a></li>
+      <li class="search-pagerItem"><a href="javascript:void(0);">2</a></li>
+      <li class="search-pagerItem"><a href="javascript:void(0);">3</a></li>
+      <li class="search-pagerItem"><a href="javascript:void(0);">4</a></li>
+      <li class="search-pagerItem"><a href="javascript:void(0);">5</a></li>
+      <li class="search-pagerItem"><a href="javascript:void(0);">6</a></li>
+      <li class="search-pagerItem"><a href="javascript:void(0);">7</a></li>
+      <li class="search-pagerItem"><a href="javascript:void(0);">8</a></li>
+      <li class="search-pagerItem"><a href="javascript:void(0);">9</a></li>
+      <li class="search-pagerItem"><a href="javascript:void(0);">10</a></li>
+    </ul>
```

Comment:
Actually, now that you bring up the pager...  I kinda wonder if it makes sense to just follow the lead from resource center, and use the default pager out of the box. 

![image](https://user-images.githubusercontent.com/5263371/28120528-e8aacb3c-66dd-11e7-8d7d-28b6f067e27f.png)

I know we're all about the maintainable approach on things, but I think we also should consider consistency, which we won't get with this approach. 

Just my two cents.  I'm not married to it one way or another, but just thought I'd offer up that feedback.